glide.yaml

```diff
@@ -7,3 +7,5 @@ import:
 - package: github.com/spf13/viper
 - package: github.com/justinas/alice
   version: ^1.0.0
+- package: github.com/stretchr/testify
```

Comment:
Maybe add testify as a testImport dependency? This way it can be ignored by the production build script. https://glide.readthedocs.io/en/latest/glide.yaml/